### ISSUE TITLE
USWDS - Bug: Gradient behind SVG images fails axe DevTools Checker

### ISSUE THREAD

**Issue by mahoneycm** [uswds/uswds#4966]
### Describe the bug

Running axe DevTools accessibility check results in a failure where background svg's are used. The failure is related to the transparent linear-gradient placed behind the svg because it cannot determine a background color for color contrast testing.

Based on the [comment](https://github.com/uswds/uswds/blob/1db0e68d7e5cf21ae52f2d8a29d5963c7aa1b681/packages/uswds-core/src/styles/mixins/general/add-background-svg.scss) before the `add-background-svg` mixin, this linear-gradient implementation was used as a trick to find browsers that don't support svg's in background-image's, and is to be used as a fallback to a png. 

>  If a browser supports both multiple backgrounds and linear gradients, it supports SVG too. So here we declare the SVG with a completely transparent linear-gradient. If that fails in an older browser, the fallback declared above will kick in.  [^1]

From my experience, there are no fallback png's being used in tandem with these background svg's. Also, USWDS no longer supports IE 11 so this functionality is no longer required.

During testing, removing the linear gradient completely does not affect the display of the icon.

[^1]: https://css-tricks.com/a-complete-guide-to-svg-fallbacks/ 

### Steps to reproduce the bug

1. View any component that implements an svg background-image for an icon, such as [accordion](https://designsystem.digital.gov/components/accordion/)
2. Run axe DevTools
3. See "Elements must have sufficient color contrast" error.
4. In the "To solve this problem, you need to..." section it reads: 
> "Element's background color could not be determined due to a background gradient"

### Expected Behavior

There should be no failure related to the `background-image`

### Related code

Output CSS from devtools:

```css
background-image: url(../img/usa-icons/add.svg),linear-gradient(transparent,transparent);
```

`add-background-svg` mixin:

```css
@mixin add-background-svg($image-name, $image-path: $theme-image-path) {
  background-image: url("#{$image-path}/#{$image-name}.svg"),
    linear-gradient(transparent, transparent);
  background-repeat: no-repeat;
}
```

### Screenshots

Example of removing the linear-gradient with no change to the icon.

https://user-images.githubusercontent.com/25211650/193071257-092d3891-670d-4eaf-9fe1-139588c1c153.mov

### System setup

- [USWDS Accordion page](https://designsystem.digital.gov/components/accordion/)
- Axe DevTools: axe-core 4.4.2
- MacOS: 12.3.1
- Google Chrome

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [X] I checked the [current issues](https://github.com/uswds/uswds/issues?q=is%3Aissue+is%3Aopen+label%3A%22Type%3A+Bug%22) for duplicate bug reports.

**Comment by amyleadem** [comment]
@mahoneycm 
I ran axe DevTools on the [accordion storybook preview](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/develop/iframe.html?args=&id=components-accordion--default&viewMode=story) and found that 5 items were flagged as "Needs review" because "Element's background color could not be determined due to a background gradient":

![image](https://user-images.githubusercontent.com/93996430/200350363-a1d1443a-6f06-4582-b85a-44742f1deb0d.png)

Manual checks revealed that these items met acceptable contrast ratios so they should not be considered errors. 

Because the `linear-gradient` fallback is likely unnecessary, I experimented with removing `linear-gradient` from `usa-accordion__button`. However, the element was still flagged as "Needs review" by Axe because "Element's background color could not be determined due to a background image": 

![image](https://user-images.githubusercontent.com/93996430/200351244-b8655d19-5a43-44a3-b6a2-ea3dfb1d4f20.png)

Because the item will continue to be flagged for review regardless of whether the `linear-gradient` is included or not, I recommend that we do not make any changes at this time. 

Please let me know if you see anything different or if you have any questions or comments.

**Comment by mahoneycm** [comment]
@amyleadem I found the same results. Since it's not an outright failure and the color contrast does in fact pass, I agree that we can ignore this.

**Comment by danny-englander** [comment]
I've run into this issue as well. I'd like to point out that some sights get strict testing where everything really needs to pass so I'd love to see a solution to this eventually.

**Comment by mahoneycm** [comment]
@danny-englander Are you using axe-core to test as well? If so, it shouldn't be throwing an actual error, just stating it needs a manual check. Axe currently does not have the availability to check color contrast with elements overlapping background images. I'm not sure if that helps with your situation. 

This issue on the axe-core repo discusses it a bit further: [https://github.com/dequelabs/axe-core/issues/3390](https://github.com/dequelabs/axe-core/issues/3390)

**Comment by danny-englander** [comment]
@mahoneycm In this case, I have been using the Axe DevTools plugin for Chrome. I see now that my issue is the same as what Amy posted above. 

<img width="1240" alt="2022-11-18_11-10-34" src="https://user-images.githubusercontent.com/609623/202750164-90434544-5a96-41a1-8407-fae5ea127512.png">

**Comment by mahoneycm** [comment]
Depending on your site's requirements regarding all accessibility tests needing to pass, that notification is technically not a failure. It pretty much just says the tool can't grant it a pass or fail automatically and needs a manual test for color contrast.

It's mostly a problem with the limitations of axe and not actually an accessibility issue. The color contrast in the case of the USWDS accordion passes color contrast requirements.
